.circleci/config.yml

```diff
@@ -22,8 +22,6 @@ workflows:
           app_name: "promxy-app"
           app_namespace: "monitoring"
           app_collection_repo: "shared-app-collection"
-          user_configmap_name: "management-cluster-promxy-config"
```

Comment:
`user_configmap_name` and `user_configmap_path` will be made obsolete by #54 anyways.

Reply:
Could you remove the file under config/promxy.yaml as well?